apps/emqx_durable_storage/src/emqx_ds_storage_skipstream_lts.erl

```diff
@@ -230,6 +236,7 @@ commit_batch(
 ) ->
     {ok, Batch} = rocksdb:batch(),
     try
+        %% is this dummy?
```

Comment:
?

Reply:
emqx_ds_lts:trie_update at L248 also persist it.

Reply:
No this is not dummy. See definition of PersistCallback: 

```
    fun(Key, Val) ->
        push_lts_persist_op(Key, Val), %% adds op to the process dictionary
        ok
    end,
```

It merely adds operations to RAM "log", from where they are added to the cooked batch. This 2-stage mechanism was added so RAFT leader doesn't corrupt its DB if leadership changes amidst processing a batch.